src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacNumericValidator.java

```diff
@@ -329,21 +340,29 @@ private QuantityValidationResponse validateWithoutUnits(final IsaacNumericQuesti
      *
      * @param trustedValue               - first number
      * @param untrustedValue             - second number
-     * @param significantFiguresRequired - the number of significant figures to perform comparisons to
+     * @param significantFiguresRequired - the number of significant figures to perform comparisons to (can be null,
+     *                                   to skip significant figure checks)
      * @return true when the numbers match
      * @throws NumberFormatException - when one of the values cannot be parsed
      */
     private boolean numericValuesMatch(final String trustedValue, final String untrustedValue,
-                                       final int significantFiguresRequired) throws NumberFormatException {
+                                       final Integer significantFiguresRequired) throws NumberFormatException {
         log.debug("\t[numericValuesMatch]");
         double trustedDouble, untrustedDouble;
 
         String untrustedParsedValue = reformatNumberForParsing(untrustedValue);
         String trustedParsedValue = reformatNumberForParsing(trustedValue);
 
-        // Round to N s.f. for trusted value
-        trustedDouble = roundStringValueToSigFigs(trustedParsedValue, significantFiguresRequired);
-        untrustedDouble = roundStringValueToSigFigs(untrustedParsedValue, significantFiguresRequired);
+        if (significantFiguresRequired == null){
```

Comment:
Our old colleague who implemented all the style rules always preferred nulls first in comparisons: `null == ...` or `null != ...`, since it was then a syntax error if you missed an equals sign because you can't assign to null. I like it as a habit, even if I think our tooling would spot the mistake if it happened.

Reply:
I did wonder about that, nice idea.